model/src/main/kotlin/utils/Extensions.kt

```diff
@@ -27,48 +27,67 @@ import org.ossreviewtoolkit.clients.clearlydefined.Provider
 import org.ossreviewtoolkit.clients.clearlydefined.SourceLocation
 import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.Package
+import org.ossreviewtoolkit.model.PackageProvider
 import org.ossreviewtoolkit.model.RemoteArtifact
 import org.ossreviewtoolkit.model.TextLocation
+import org.ossreviewtoolkit.model.VcsInfo
 import org.ossreviewtoolkit.model.VcsInfoCurationData
 import org.ossreviewtoolkit.utils.common.percentEncode
 
 internal fun TextLocation.prependPath(prefix: String): String =
     if (prefix.isBlank()) path else "${prefix.removeSuffix("/")}/$path"
 
 /**
- * Map an [Package] to a ClearlyDefined [ComponentType] and [Provider]. Note that an
- * [identifier's type][Identifier.type] in ORT currently implies a default provider. Return null if a mapping is not
- * possible.
+ * Map a [Package] [identifier's type][Identifier.type] to a ClearlyDefined [ComponentType]. Return null if a mapping is
```

Comment:
maybe instead: "Map [type][Identifier.type] of a [package identifier] [Package.id] to [..]"